.gitmodules

```diff
@@ -22,3 +22,7 @@
     path = external/MonoTouch.Dialog
     url = ../../migueldeicaza/MonoTouch.Dialog
     branch = dotnet
+[submodule "api-tools"]
+	path = external/api-tools
+	url = ../../rolfbjarne/api-tools
```

Comment:
My thinking was to let it mature outside of xamarin for a while to make sure how it works out, although I don't really care either way. We've basically been using Sebastien's repos since forever for instance.

Maybe @dalexsoto has an opinion?

Reply:
It's trivial to re-fork if Rolf wins the lottery or something.

Reply:
I think it is fine to keep it under @rolfbjarne's repo for now, we can alway bring it over once we feel it is the time